lib/opensusebasetest.pm

```diff
@@ -443,6 +443,9 @@ sub export_logs {
     my $compression = is_sle('=12-sp1') ? 'bz2' : 'xz';
     script_run "save_y2logs /tmp/y2logs_clone.tar.$compression";
     upload_logs "/tmp/y2logs_clone.tar.$compression";
+    script_run("zypper -n patch --debug-solver --with-interactive -l");
+    script_run("tar -cvjf /tmp/solverTestCase.tar.bz2 /var/log/zypper.solverTestCase/*");
+    upload_logs "/tmp/solverTestCase.tar.bz2 ";
```

Comment:
The generic `export_logs` already takes very long and the data is growing impacting how much jobs we can store. I don't think this should be done in all the cases. How about you set a flag `IN_ZYPPER_CALL` in zypper_call and unset when leaving again. Then evaluate that flag here

Reply:
> 
> 
> The generic `export_logs` already takes very long and the data is growing impacting how much jobs we can store. I don't think this should be done in all the cases. How about you set a flag `IN_ZYPPER_CALL` in zypper_call and unset when leaving again. Then evaluate that flag here

Good idea, i will try it.

Reply:
@okurz  updated .